src/net/RTP/VideoStream.cs

```diff
@@ -203,9 +207,18 @@ public void SendH265Frame(uint durationRtpUnits, int payloadID, byte[] sample)
             if(CheckIfCanSendRtpRaw())
             {
                 var nals = H265Packetiser.ParseNals(sample);
-                var aggregatedNals = H265Packetiser.CreateAggregated(nals, RTPSession.RTP_MAX_PAYLOAD);
-                foreach (var nal in aggregatedNals)
+
+                // aggregation is only on 2 or more small nals
+                if (nals.Where(x => x.NAL.Length < RTPSession.RTP_MAX_PAYLOAD).Count() > 1)
+                {
+                    //logger.LogTrace("(ou) Trying aggregating {nals} nals", nals.Count());
+                    nals = H265Packetiser.CreateAggregated(nals, RTPSession.RTP_MAX_PAYLOAD);
+                }
```

Comment:
`Any` will only cause the `nals` sequence to be iterated until the first is found, while `Count` will cause the `nals` sequence to be completely iterated.

Both incur in delegate allocation (although in the latest SDKs this may be a static delegate instance) and invocation.

I would prefer something like:

``csharp
                foreach (var nal in nals)
                {
                    if (nal.NAL.Length< RTPSession.RTP_MAX_PAYLOAD)
                    {
                        nals = H265Packetiser.CreateAggregated(nals, RTPSession.RTP_MAX_PAYLOAD);
                        break;
                    }
                }
```